site/content/docs/main/how-velero-works.md

```diff
@@ -77,7 +80,8 @@ This allows restore functionality to work in a cluster migration scenario, where
 
 Likewise, if a backup object exists in Kubernetes but not in object storage, it will be deleted from Kubernetes since the backup tarball no longer exists.
 
-[10]: hooks.md
+[10]: backup-hooks.md
+[11]: restore-hooks.md
```

Comment:
Why completely separate pages instead of separate sections on one page?

Reply:
Created separate pages to prevent the `hooks.md` from becoming a long page and to make this content easily discoverable. Also, there isn't much common between backup and restore hooks other than them just being extension points to Velero.